taf/repository_tool.py

```diff
@@ -592,6 +662,11 @@ def get_role_repositories(self, role, parent_role=None):
             if all([fnmatch(repo, path) for path in role_paths])
         ]
 
+    def get_delefations_info(self, role_name):
+        # load repository is not already loaded
+        self._repository
```

Comment:
The problem is that that would prevent us from instantiating an empty `AuthRepo` because `tuf` raises an error if mandatory metadata files do not exist yet.